lib/ansible/modules/system/filesystem.py

```diff
@@ -23,9 +23,11 @@
 version_added: "1.2"
 options:
   fstype:
+    choices: [ "ext4", "ext4dev", "ext3", "ext2", "xfs", "btrfs", "reiserfs", "lvm"]
     description:
     - File System type to be created.
     - reiserfs support was added in 2.2.
+    - lvm support was added in 2.4.
```

Comment:
2.5 at this point but i'll fix after merge

Reply:
@bcoca You could push another commit to this PR to fix that before merging.

Reply:
It needs to be retested by CI anyways since the CI result is stale.

Reply:
@mattclay rebuild_merge is supposed to make the bot rerun CI and only merge if it succeeds